### PR TITLE
Add more details about cluster-driver-registrar

### PR DESCRIPTION
**Pull request by msau42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
 [![Build Status](https://travis-ci.org/kubernetes-csi/driver-registrar.svg?branch=master)](https://travis-ci.org/kubernetes-csi/driver-registrar)
 # Cluster Driver Registrar
 
-A sidecar container that
+A controller sidecar container that
```
Comment (davidz627) [inline README.md]: cluster-wide controller (?)

File: README.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 
 1. Registers the containerized CSI driver with Kubernetes and create a CSIDriver object
```
Comment (davidz627) [inline README.md]: what does "Registering the CSI Driver" mean in this context

File: README.md [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
 
+This sidecar container is only needed if you need the following Kubernetes
+features:
```
Comment (davidz627) [inline README.md:11]: maybe add a link on how to "activate/use" these features if the user reading this is interested

Comment (msau42) [reply README.md:11]: Added a link to the documentation below.  Eventually, we'll want to move this documentation right into this repo and link to it from the docs.

File: README.md (hunk 1 above)
Comment (msau42) [reply README.md]: I called it "cluster-level"

File: README.md (hunk 2 above)
Comment (msau42) [reply README.md]: Removed and just said that it creates the CSIDriver object

**Pull request merged**